src/tactic/ui/table/task_element_wdg.py

```diff
@@ -433,6 +433,13 @@ def alter_task_search(my, search, prefix='children', prefix_namespace='' ):
     def preprocess(my):
```

Comment:
combine the import statement
from pyasm.common import Environment, TacticException